<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Definire</class>
 <widget class="QDialog" name="Definire">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="definitedeja">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>761</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Strategia companiei a fost definita, se pot aduce modificari asupra acesteia in tabelul de mai jos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inchide">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>550</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Salveaza si inchide</string>
   </property>
  </widget>
  <widget class="QLabel" name="trebuiedef">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Strategia companiei trebuie definita </string>
   </property>
  </widget>
  <widget class="QLabel" name="defmisiune">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Definirea misiunii:</string>
   </property>
  </widget>
  <widget class="QLabel" name="defviziune">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Definirea viziunii:</string>
   </property>
  </widget>
  <widget class="QLabel" name="defvalori">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Definirea valorilor:</string>
   </property>
  </widget>
  <widget class="QLabel" name="defstr">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Definirea strategiei pentru o dezvoltare durabila:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="misiuneEdit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>61</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="viziuneEdit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>90</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="valoriEdit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>120</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="strEdit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>150</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="salveaza">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>500</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Salveaza</string>
   </property>
  </widget>
  <widget class="QPushButton" name="anuleaza">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>550</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Anulare</string>
   </property>
  </widget>
  <widget class="QLabel" name="ghid">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ghid pentru completarea campurilor de mai sus:</string>
   </property>
  </widget>
  <widget class="QLabel" name="ghid2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>791</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QLabel" name="ghid3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>791</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>781</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <zorder>tableView</zorder>
  <zorder>definitedeja</zorder>
  <zorder>inchide</zorder>
  <zorder>trebuiedef</zorder>
  <zorder>defmisiune</zorder>
  <zorder>defviziune</zorder>
  <zorder>defvalori</zorder>
  <zorder>defstr</zorder>
  <zorder>misiuneEdit</zorder>
  <zorder>viziuneEdit</zorder>
  <zorder>valoriEdit</zorder>
  <zorder>strEdit</zorder>
  <zorder>salveaza</zorder>
  <zorder>anuleaza</zorder>
  <zorder>ghid</zorder>
  <zorder>ghid2</zorder>
  <zorder>ghid3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
